eng/performance/sdk_scenarios.proj

```diff
@@ -41,86 +41,108 @@
   
   <ItemGroup>
     <SDKWorkItem Include="@(Framework -> 'SDK Console Template %(Identity)')" Exclude="*2.1;*3.0;*3.1;*5.0;*6.0">
-      <PayloadDirectory>$(ScenariosDir)emptyconsoletemplate</PayloadDirectory>
+      <ScenarioDirectoryName>emptyconsoletemplate</ScenarioDirectoryName>
+      <PayloadDirectory>$(ScenariosDir)%(ScenarioDirectoryName)</PayloadDirectory>
```

Comment:
I am not sure if it will work, but can this PayloadDirectory definition be moved into an ItemDefinitionGroup so that it doesn't need to be copy-pasted everywhere?

Reply:
That will be part of bigger refactoring I'm doing in #2665.